apple/RNCWebView.m

```diff
@@ -441,39 +441,37 @@ - (void)didMoveToWindow
     
     [self setBackgroundColor: _savedBackgroundColor];
     
-    if (!reusedWebViewInstance) {
```

Comment:
Other than removing this `if (!reusedWebViewInstance)` check, the other changes in this PR are all an indentation change.